tests/plugins/issue-archiver/index.js

```diff
@@ -35,6 +35,7 @@ describe("issue-archiver", () => {
 
         nock("https://api.github.com")
             .get("/installation/repositories")
+            .query(true)
```

Comment:
What does this do?

Reply:
it matches against all query types... in this case it had `pageSize=100` (or something like that) and test were failing on my machine.
This technique has been used in above mock also.